cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffCommand.java

```diff
@@ -213,7 +255,37 @@ public void execute() throws CommandException {
     }
   }
 
-  void executeUnsafe() throws CommandException {
+  private void checkMaxStringsBlock(ExtractionDiffPaths extractionDiffPaths)
+      throws MissingExtractionDirectoryException, CommandException {
+    if (!this.skipMaxStringsBlock && !Strings.isNullOrEmpty(this.pushToBranchName)) {
+      ExtractionDiffStatistics extractionDiffStatistics =
+          this.extractionDiffService.computeExtractionDiffStatistics(extractionDiffPaths);
+      if (this.maxStringsAddedBlock != null
+          && extractionDiffStatistics.getAdded() > this.maxStringsAddedBlock) {
+        this.notificationSender.ifPresent(
+            notificationSender ->
+                notificationSender.sendMessage(
+                    String.format(
+                        "The branch: %s has more than %d strings added",
```

Comment:
@DarKhaos - one last ask - could we modify these notification messages (for both notif sender and the exception) for both types of blocks (add/removals) to also mention the impact? 

I.e.: "Error: The branch ..  has more than .. added/removed. The changes will not be sent to Mojito as a result. If this a valid and expected change, decrease or disable the block limit from the current value of: X."

@maallen - if you have a better idea of how to phrase the above to explain the impact to the end user

Reply:
Could that message be: "Error: The branch .. has more than .. added/removed. The changes will not be sent to Mojito as a result. If this a valid and expected change, apply the skip-max-strings-block label to the PR"?

Reply:
I don't think we should mention labels in the error message as it will be possible for someone not using this integration to utilise this blocking feature.

I think @garionpin's message is sufficient, it describes the issue and provides resolution steps. From our perspective, in this particular incidents I would want our team to be the people who apply the label rather than the individual who raised the PR, so I think we should avoid mentioning it in the error message.